tests/fsharp/TestResults/_pkrvmq0rgcvqdmg_2025-07-21_15_41_19.trx

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Also remove this file.
You are still commiting it.


At last, create a release notes entry for this change as the comment in the PR suggests.
Follow the style of existing entries.